wrapanapi/msazure.py

```diff
@@ -477,9 +477,14 @@ def remove_nics_by_search(self, nic_template, resource_group=None):
                                        resource_group=resource_group or self.resource_group)
 
         for nic in nic_list:
-            operation = self.network_client.network_interfaces.delete(
-                resource_group_name=resource_group or self.resource_group,
-                network_interface_name=nic)
+            try:
+                operation = self.network_client.network_interfaces.delete(
+                    resource_group_name=resource_group or self.resource_group,
+                    network_interface_name=nic)
+            except CloudError as e:
+                self.logger.info('"%s" nic can\'t be removed', nic)
```

Comment:
Can you log `e.error.error` here as well, and use 'logger.error' instead of info